packages/react/src/Card/Card.stories.tsx

```diff
@@ -0,0 +1,212 @@
+import { Meta, Story } from '@storybook/react';
+import { EyeIcon, MoreVerticalIcon } from '@mezzanine-ui/icons';
+import Icon from '../Icon';
+import Card, { CardProps } from './Card';
+import CardActions from './CardActions';
+import { TypographyProps } from '../Typography';
+
+export default {
+  title: 'Data Display/Card',
+} as Meta;
+
+type PlaygroundArgs = Required<CardProps>;
```

Comment:
Remove object like props(ex: titleProps). If there is anything you want to demo(ex: titleProps.variant), write your own args

Reply:
![image](https://user-images.githubusercontent.com/83571075/118266472-3be45280-b4ed-11eb-9f04-66f6d69240cc.png)